infrastructure/k8s/helm-chart/eoapi/templates/db/manifest.yaml

```diff
@@ -1,9 +1,12 @@
+{{- include "eoapi.validateValues" . }}
```

Comment:
reviewer comment: this is how we trigger validation for required, non-empty `values.yaml` validation

Reply:
@ranchodeluxe interesting.

I've been liking the approach of using a Schema JSON file for values like https://www.arthurkoziel.com/validate-helm-chart-values-with-json-schemas/ - is there a benefit of using these custom rules versus using a JSON schema?

(This is definitely fine and this all looks GREAT, just curious since we've generally been taking the approach of using JSON schemas to validate values elsewhere).

Reply:
Oh nice, can you link to examples of how you're using this in DS repos? I copied this from Bitnami helm charts

Via json schemas, where does the validation show up when it fails?

Reply:
I think the reason Bitnami might be using this is so it can run multiple validators and have cleaner output @batpad (see output below from a failed run). Would love to see what Schema JSON file does in comparison

```python
$ helm template . -s templates/vector/manifest.yaml -f values.yaml
Error: execution error at (eoapi/templates/db/manifest.yaml:1:4): 
VALUES VALIDATION:
DB_SETTINGS:
  PGUSER: "" <<<
  `PGUSER` is required and should be a non-empty string
DB_SETTINGS:
  POSTGRES_USER: "" <<<
  `POSTGRES_USER` is required and should be a non-empty string
DB_SETTINGS:
  PGPASSWORD: "" <<<
  `PGPASSWORD` is required and should be a non-empty string
DB_SETTINGS:
  POSTGRES_PASSWORD: "" <<<
  `POSTGRES_PASSWORD` is required and should be a non-empty string
gitSha: "" <<<
  `gitSha` is required and should be a non-empty string
```

Reply:
The other nice thing about the values schema JSON is you can pretty easily generate simple docs from it, example for the above: http://devseed.com/lulc-infra/

Reply:
I'm down for using JSON Schema for validation. I created a ticket b/c unfortunately today and tomorrow I have to put more time on other work: https://github.com/developmentseed/eoAPI/issues/88